src/main/java/space/npstr/magma/events/audio/conn/Shutdown.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2018 Dennis Neufeld
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package space.npstr.magma.events.audio.conn;
+
+/**
+ * Created by napster on 21.06.18.
+ */
+public class Shutdown implements ConnectionEvent {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a private constructor to hide the implicit public one. ([squid:S1118](https://sonarcloud.io/organizations/napstr-github/rules#rule_key=squid%3AS1118))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=space.npstr.magma%3Amagma&pullRequest=4&issues=AWQknaZK_bEzQ-kUd4DA&open=AWQknaZK_bEzQ-kUd4DA)

[ISSUE_KEY]: AWQknaZK_bEzQ-kUd4DA